dashboards/google-cloud-tasks-queue/metadata.yaml

```
@@ -0,0 +1,10 @@
+sample_dashboards: 
+  - 
+    category: Cloud Tasks Queue
+    id: cloud-tasks-queue
+    display_name: Cloud Tasks
+    description: |-
+      This dashboard has charts displaying Cloud Tasks Queue metrics.
```

```suggestion
      This dashboard has charts displaying Task Queue metrics.
```

I think there are two products - Cloud Tasks and Task Queues - can we clarify which this dashboard is showing metrics for and align on one name here?